Assets/MRTK/SDK/Experimental/PinchSliderBox/PinchSliderBox.cs

```diff
@@ -0,0 +1,499 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using static Microsoft.MixedReality.Toolkit.UI.PinchSlider;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// Manages and creates sliders to allow for non-uniform scaling of a box along multiple axes.
+    /// The [MinMaxScaleConstraint](xref:Microsoft.MixedReality.Toolkit.UI.MinMaxScaleConstraint) is 
```

Comment:
We probably want to adjust this to the correct format for rendering in VS. Our docs generation will convert it.

```suggestion
    /// The <see cref="MinMaxScaleConstraint"/> is 
```